src/main/java/uk/nhs/digital/nhsconnect/nhais/service/edifact_to_fhir/AmendmentTransactionMapper.java

```diff
@@ -0,0 +1,19 @@
+package uk.nhs.digital.nhsconnect.nhais.service.edifact_to_fhir;
+
+import org.hl7.fhir.r4.model.Parameters;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.ReferenceTransactionType;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.Transaction;
+
+@Component
+public class AmendmentTransactionMapper implements TransactionMapper {
+    @Override
+    public void map(Parameters parameters, Transaction transaction) {
+        //TODO: to be implemented
+    }
+
+    @Override
+    public ReferenceTransactionType.TransactionType getTransactionType() {
+        return ReferenceTransactionType.TransactionType.AMENDMENT;
```

Comment:
This refers to an outbound (GP->HA) amendment (AMG) type G2.

It should be an inbound (HA->GP) amendment (AMF) type F1.

We'll need to update TransactionType to differentiate between inbound and outbound transactions.